services/workflows-service/src/env.ts

```diff
@@ -1,3 +1,32 @@
-// Identifiers to pass into NestJS's ConfigService
+import 'dotenv/config';
+import { createEnv } from '@t3-oss/env-core';
+import { z } from 'zod';
 
-export const JWT_EXPIRATION = 'JWT_EXPIRATION';
+export const env = createEnv({
+  /*
+   * clientPrefix is required.
+   */
+  clientPrefix: 'PUBLIC_',
```

Comment:
What `clientPrefix` means in that context?

Reply:
It's for frameworks like NextJS where only env variables prefixed with PUBLIC_ are available client-side.